README.md

```diff
@@ -621,7 +619,10 @@ echo "Operation status: ".$status->getStatus()."<br />";
 ```
 
 ##Media Services
-
+> **Note**
+> 
+>The current Media Services API only works with PHP 5.5 and above.
```

Comment:
yes, I can change the calls to empty() or create a wrapper for empty that is compatible with 5.4. There may be other things in the code not compatible with 5.4, too. I will try to fix them a bit later.